libtiledbsoma/src/tiledbsoma/tiledbsoma

```diff
@@ -33,15 +33,7 @@
 #ifndef __TILEDBSOMA__
 #define __TILEDBSOMA__
 
-// TODO Uncomment after finishing Python and R bindings
-// #include "soma_collection.h"
-// #include "soma_dataframe.h"
-// #include "soma_dense_ndarray.h"
-// #include "soma_experiment.h"
-// #include "soma_measurement.h"
-// #include "soma_object.h"
-// #include "soma_sparse_ndarray.h"
-
+#include "tiledbsoma_export.h"
```

Comment:
I presume that is a cmake generated file?  Should we say so?  A bit weird to include something not in the sources.

Reply:
I will add a comment. We need this file to support exporting symbols in a cross-platform fashion (replacing [this](https://github.com/single-cell-data/TileDB-SOMA/blob/fe829ad4a9669f48c12a5198a96cea699a174231/libtiledbsoma/src/soma/logger_public.h#L44)). The Core already has one.

Reply:
Understood from what's in it but I think I have seen many other project have such `#define` statements in the normal include directories, no?

Reply:
Thanks for dealing with the Windows build by the way.  Much appreciated.

Reply:
> I have seen many other project have such `#define` statements in the normal include directories

We could do that, but having CMake generate the file automatically handles the varying platforms, compilers and static or dynamic linkage.